samples/ChatbotExamplev2/Bots/PersonalizerChatbot.cs

```diff
@@ -1,20 +1,21 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using LuisBot.Model;
-using LuisBot.ReinforcementLearning;
 using Microsoft.Azure.CognitiveServices.Personalizer;
 using Microsoft.Azure.CognitiveServices.Personalizer.Models;
 using Microsoft.Bot.Builder;
 using Microsoft.Bot.Schema;
 using Newtonsoft.Json;
+using ChatbotSample.Model;
+using ChatbotSample.ReinforcementLearning;
+using ChatbotSample.Services;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.BotBuilderSamples
+namespace ChatbotSample.Bots
```

Comment:
For my own education: Why did we remove `Microsoft` from the namespace?

Reply:
This goes back to how the original chatbot sample was created, which was as a modification of this since outdated sample in the Microsoft BotBuilderSamples repo: https://github.com/jwiley84/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/12.nlp-with-luis (using a link from a fork since that sample is now gone). When we originally created this sample off of that one, it looks like we just kept all the same namespaces. I created a new project "from scratch" (ish) so I didn't preserve the old namespaces